docker/amppkg.example.toml

```diff
@@ -0,0 +1,129 @@
+# Example toml using the fake testdata certs shipped with AMP Packager.
+#
+# The signed exchanges generated by this are NOT valid and will NOT work with
+# any browser or AMP cache!
+
+
+# This is a TOML 0.4.0 file, as specified by https://github.com/toml-lang/toml.
```

Comment:
Maybe just say "see ../amppkg.example.toml for details" instead of duplicating the comments here. Otherwise, they'll quickly get out of date.

Reply:
Done (wait for next push).